<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cadastro</class>
 <widget class="QMainWindow" name="Cadastro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 237);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-20</y>
      <width>951</width>
      <height>721</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>951</width>
      <height>721</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>951</width>
      <height>721</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.534, y1:0, x2:0.523, y2:1, stop:0 rgba(0, 101, 154, 203), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>371</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.185, fx:0.5, fy:0.5, stop:0 rgba(0, 104, 214, 214), stop:1 rgba(0, 226, 228, 228));

border-radius: 30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="CadastroNome">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>271</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Agency FB&quot;;
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Nome do Produto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CadastroCodigo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>271</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Agency FB&quot;;
border-radius: 15px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Codigo do Produto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CadastroPreco">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>450</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>271</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 16pt &quot;Agency FB&quot;;
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Pre√ßo do Produto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CadastroCadastrar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>500</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(255, 255, 255);
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>CADASTRAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>501</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1.185, fx:0.5, fy:0.5, stop:0 rgba(0, 104, 214, 214), stop:1 rgba(0, 226, 228, 228));
font: 36pt &quot;Agency FB&quot;;
border-radius:25px;
</string>
     </property>
     <property name="text">
      <string>                    Cadastro</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Voltar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-width: 1px;
border-color: rgb(0, 51, 102);
background-color: rgb(255, 255, 255);
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>Voltar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>91</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
border-style: solid;
border-radius: 40px;
border-width: 2px;
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>fotos/sell-stock.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>-90</x>
       <y>190</y>
       <width>1331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>-70</x>
       <y>350</y>
       <width>1331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>-90</x>
       <y>420</y>
       <width>1331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>500</y>
       <width>1331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>-230</x>
       <y>280</y>
       <width>1331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_18</zorder>
    <zorder>label_17</zorder>
    <zorder>label_13</zorder>
    <zorder>label_16</zorder>
    <zorder>label_14</zorder>
    <zorder>label_9</zorder>
    <zorder>CadastroNome</zorder>
    <zorder>CadastroCodigo</zorder>
    <zorder>CadastroPreco</zorder>
    <zorder>CadastroCadastrar</zorder>
    <zorder>label_10</zorder>
    <zorder>Voltar</zorder>
    <zorder>label</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="fotos.qrc"/>
 </resources>
 <connections/>
</ui>
